api/src/catalog/catalog.utils.ts

```diff
@@ -92,8 +94,16 @@ const exists = (p: string) =>
 function transformListingToSitemapListing(
   listing: CatalogBySeasonQuery['listings'][number],
 ): SitemapListing {
+  const year = parseInt(listing.season_code.substring(0, 4), 10);
+  const season = listing.season_code.substring(4);
+
+  let lastmod = '';
+  if (season === '03' || season === '02') lastmod = `${year}-01-01`;
```

Comment:
Courses have never been published on 01-01; Maybe 04-01 is more realistic?

We can even choose roughly when the semesters start (01-15, 05-25, 08-25) because courses before this date usually change a lot.

Reply:
ya but im not sure we can list it as a course in the future like for the fall and spring 2024 classes we dont want those last mod in the future

Reply:
What about "semester start date vs. today, whichever is earlier"?

Reply:
ya but then we want it constantly updating? i guess thats not horrible tbh....

Reply:
Yeah doesn't sound like a bad idea since we only do this at most once a day anyway